src/Webflow.js

```diff
@@ -76,6 +76,7 @@ export default class Webflow {
         method,
         headers: this.headers,
         mode: "cors",
+        ...fetchOpts,
```

Comment:
One thing about this approach is it would override the headers (including auth) if someone added a header entry.

Reply:
Hmmm. Could change the order:

```
mode: "cors",
...fetchOpts,
method,
headers: this.headers,
```

I think this would do the trick. Thoughts?